front/src/modules/ui/button/components/IconButtonGroup.tsx

```diff
@@ -14,21 +15,24 @@ const StyledIconButtonGroupContainer = styled.div`
 type IconButtonGroupProps = {
   variant: IconButtonVariant;
   size: IconButtonSize;
-  children: React.ReactElement[];
-};
+  children: React.ReactElement | React.ReactElement[];
+} & Omit<ComponentProps<'div'>, 'children'>;
```

Comment:
Done. I kept the `Omit<..., 'children'>` though, otherwise `IconButtonGroupProps['children']`'s type would then become `ReactNode & (React.ReactElement | React.ReactElement[])`.